MSAL/src/requests/MSALBaseRequest.m

```diff
@@ -398,6 +400,19 @@ - (NSString *)claims
                                                            developerClaims:_parameters.decodedClaims];
 }
 
+- (NSString *)enrollmentId
```

Comment:
I suggest moving the whole method to MSIDAuthority. Base implementation would try to find `enrollmentId` in the cache when MSIDADFSAuthority will return nil.